charts/reform-scan-blob-router/values.yaml

```diff
@@ -28,6 +28,8 @@ java:
     FLYWAY_USER: blob_router@reform-scan-blob-router-{{ .Values.global.environment }}
     FLYWAY_SKIP_MIGRATIONS: true
 
+    DELETE_DISPATCHED_FILES_CRON: "0 0/10 * * * *"
```

Comment:
looks extremely rare, considering the main job will run every ~30sec.

Reply:
In test envs it runs every 10 seconds. In production, what's the point of making it run any more frequently than once in 10 minutes?

Reply:
We could even, perhaps, delete files at the end of the day. This way we would be able to resend the files in case our client loses them.

Reply:
Plus, the main job shouldn't run every 30 seconds, but approx. 10 minutes, as well. Otherwise, each file would be checked for readiness by each instance approx. 60 times.

Reply:
I agree with Leszek - no point to run delete job as frequently as processing job

Reply:
Worth discussing within the team